src/App.tsx

```diff
@@ -1,26 +1,262 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { deleteTodo, getTodos, postTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { Footer } from './Components/Footer';
+import { updateTodos } from './api/todos';
+import classNames from 'classnames';
+import { TodoList } from './Components/TodoList';
```

Comment:
General convention folder that includs multiply components should be named in lowercase.
Compare import and name TodoList files.
It is a reason of failed lint and tests